src/qibo/backends/__init__.py

```diff
@@ -140,8 +141,13 @@ def _default_transpiler(cls):  # pragma: no cover
         qubits = cls._backend.qubits
         natives = cls._backend.natives
         connectivity_edges = cls._backend.connectivity
-        if qubits is not None and natives is not None and connectivity is not None:
-            natives_enum = NativeGates.from_gatelist(natives)
+        if (
+            qubits is not None
+            and natives is not None
+            and connectivity_edges is not None
+        ):
+            # natives_enum = NativeGates.from_gatelist(natives)
+            natives_enum = reduce(or_, [NativeGates[nat] for nat in natives])
```

Comment:
If you confirm the next line to be working, then remove the first one as it's just dead code.

Reply:
6c77fde